File: Problem_Set_01/cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int yo_2_cents;

    do
    {
        yo_2_cents = get_int("Cents owed: ");
    }
    while (yo_2_cents < 0);

    int centcount = 0;

    centcount += yo_2_cents / 25;
    yo_2_cents %= 25;

    centcount += yo_2_cents / 10;
    yo_2_cents %= 10;

    centcount += yo_2_cents / 5;
    yo_2_cents %= 5;

    centcount += yo_2_cents;

    printf("%i\n", centcount);
}

File: Problem_Set_06/sentimental/cash/cash.py
import cs50

while True:
    change_owed = cs50.get_float("Change owed: ")

    if change_owed > 0:
        remaining_cents = round(change_owed * 100)
        total_coins = 0

        # Coin values in cents
        QUARTER = 25
        DIME = 10
        NICKEL = 5
        PENNY = 1

        total_coins += remaining_cents // QUARTER
        remaining_cents %= QUARTER

        total_coins += remaining_cents // DIME
        remaining_cents %= DIME

        total_coins += remaining_cents // NICKEL
        remaining_cents %= NICKEL

        total_coins += remaining_cents // PENNY

        print(total_coins)
        break
